channels/de.m3u

```diff
@@ -414,9 +414,9 @@ https://www.onairport.live/sophiatv/smil:sophia-tv-en.smil/playlist.m3u8
 http://fs.live.sr.de/i/sr_universal02@107595/master.m3u8
 #EXTINF:-1 tvg-id="SRFernsehen.de" tvg-name="SR Fernsehen" tvg-country="DE" tvg-language="" tvg-logo="http://logos.zattic.com/images/channels/logos/c50695d215e17cb3f886/black/84x48.png" group-title="",SR Fernsehen (720p)
 https://srlive24-lh.akamaihd.net/i/sr_universal02@107595/master.m3u8
-#EXTINF:-1 tvg-id="Studio47NRW1800.de" tvg-name="Studio 47 (NRW - 18:00)" tvg-country="DE" tvg-language="" tvg-logo="https://i.imgur.com/xCiU1TB.png" group-title="",Studio 47 (NRW - 18:00) (576p) [Offline]
+#EXTINF:-1 tvg-id="Studio47.de" tvg-name="Studio 47" tvg-country="DE" tvg-language="" tvg-logo="https://i.imgur.com/xCiU1TB.png" group-title="",Studio 47 (576p) [Offline]
 https://5852afe96c9bb.streamlock.net/studio47-live/mp4:livestream/playlist.m3u8
```

Comment:
Works for me.

![vlcsnap-2021-08-14-00h53m30s689](https://user-images.githubusercontent.com/30985701/129425306-c1154318-4122-4689-824f-f2aba2104046.png)